src/db/database.cr

```diff
@@ -4,21 +4,20 @@ require "weak_ref"
 module DB
   # Acts as an entry point for database access.
   # Connections are managed by a pool.
-  # The connection pool can be configured from URI parameters:
+  #
+  # ## Database URI
+  #
+  # Connection parameters are configured in a URI. The format is specified by the individual database drivers. See [the book](https://github.com/crystal-lang/crystal-book/tree/master/database#open-database) for examples.
+  #
+  # `DB#open` The connection pool can be configured from URI parameters:
   #
   #   - initial_pool_size (default 1)
   #   - max_pool_size (default 0 = unlimited)
   #   - max_idle_pool_size (default 1)
   #   - checkout_timeout (default 5.0)
   #   - retry_attempts (default 1)
   #   - retry_delay (in seconds, default 1.0)
-  #
-  # When querying a database prepared statements are used by default.
-  # This can be changed from the `prepared_statements` URI parameter:
-  #
-  #   - prepared_statements = `true`|`false` (default `true`)
-  #
-  # It should be created from DB module. See `DB#open`.
+  #   - prepared_statements (bool, default true)
```

Comment:
I prefer to mention `true` or `false` directly since the value is expressed in a query string.